cartographer_ros/cartographer_ros/node.cc

```diff
@@ -459,7 +459,10 @@ void Node::FinishAllTrajectories() {
 
 void Node::FinishTrajectory(const int trajectory_id) {
   carto::common::MutexLocker lock(&mutex_);
+  CHECK(is_active_trajectory_.count(trajectory_id) > 0 &&
```

Comment:
Yes, that's exactly what I had in mind, I just forgot about it. Rebased and changed to use at().